cartridges/int_algolia/cartridge/scripts/algolia/steps/sendDeltaExportProducts.js

```diff
@@ -0,0 +1,214 @@
+'use strict';
+
+/*
+    path to the zip:
+        Impex / src / platform / outbox / <consumer> / <exportFile> / <seqNum>.zip
+
+    zip structure
+        <seqNum>.zip / <UUID> / catalogs / <catalogID> / catalog.xml
+
+    consumer - job step parameter - comma-separated values - creates deltas for multiple consumers if requested, we'll only use one value
+    exportFile - job step parameter - it's actually a folder, the CatalogDeltaExport system job step calls it a file, so we'll keep it like this for consistencies' sake
+    seqNum - six-digit zero-padded sequential number, each run of the delta export job increments it by one - looks like this: 000001, 000002, etc.
+    UUID - randomly generated by the delta export, looks like this: ebff9c4e-ac8c-4954-8303-8e68ec8b190d
+    catalogID - job step parameter - comma-separated values - creates a separate folder for each catalogID supplied
```

Comment:
The descriptions here refer to the folder structure above. At this point it's a singular `catalogID` extracted from the `catalogIDs` job parameter. I'll reword so that it's clearer.